test/source/tests/compose.ts

```diff
@@ -1684,6 +1684,22 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'check attachment after switch to forward',
+      testWithBrowser('compatibility', async (t, browser) => {
+        const appendUrl = 'threadId=18625ff9e8642033&skipClickPrompt=___cu_false___&replyMsgId=18625ff9e8642033';
+        const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility', {
+          appendUrl,
+          hasReplyPrompt: true,
+        });
+        await composePage.waitAndClick('@encrypted-reply');
+        await composePage.waitAndClick('@action-show-reply-options-popover');
+        await composePage.waitAndClick('@action-toggle-a_forward');
+        // Check attachment is still present after switch to forward
+        await composePage.waitForContent('.qq-upload-file', 'test.txt');
```

Comment:
Let's also add testing switch back to reply here, to test that attachments are removed.